go.mod

```diff
@@ -5,6 +5,7 @@ go 1.19
 require (
 	github.com/distribution/distribution/v3 v3.0.0-20210507173845-9329f6a62b67
 	github.com/emirpasic/gods v1.18.1
+	github.com/evanphx/json-patch v4.12.0+incompatible
```

Comment:
Why don't we just use `github.com/evanphx/json-patch/v5`?

Reply:
updated.